arroyo/backends/kafka/configuration.py

```diff
@@ -111,10 +199,15 @@ def build_kafka_producer_configuration(
         default_config, bootstrap_servers, override_params
     )
 
+    # Extract client.id to use as producer name for metrics
+    producer_name = broker_config.get("client.id")
```

Comment:
Where do we set this ?

Reply:
the idea is that we let the user configure rdkafka's `client.id` (since they are already responsible for coming up with a kafka producer config) and then use that as a tag.